app/(tabs)/guests.tsx

```diff
@@ -37,22 +40,32 @@ export default function App() {
     useCallback(() => {
       if (!isStale) return;
 
+      async function getUser() {
+        const response = await fetch('/api/user');
+
+        const body: UserResponseBodyGet = await response.json();
+
+        if ('error' in body) {
+          router.replace('/(auth)/login?returnTo=/(tabs)/guests');
```

Comment:
@Eprince-hub @ProchaLu We are only using `?returnTo` in the Guests and Notes tabs here

It should probably be in every place that requires authentication, eg. check:

1. Single guests screen
2. Single notes screen
3. Create guests screen
4. Create notes screen

Reply:
@ProchaLu opened 2 PRs:

- https://github.com/upleveled/expo-example-fall-2024-atvie/pull/3
- TODO: lecture notes